tests/test_content.py

```diff
@@ -421,6 +421,83 @@ def test_datetime():
     assert new_result == '2001-02-03 04:05'  # output
 
 
+def test_nested_align():
+    """
+    # Nested Formats
```

Comment:
I would even suggest not calling it "nested formats" but rather "Parametrized Formats" or something similar.